source/MaterialXRender/ShaderRenderer.h

```diff
@@ -118,7 +118,7 @@ class ShaderRenderer
 
     /// Save the current contents of the off-screen hardware buffer to disk.
     /// @param filePath Path to file to save rendered image to.
-    virtual void saveImage(const FilePath& filePath) = 0;
+    virtual void saveImage(const FilePath& filePath, ConstImagePtr image, bool verticalFlip) = 0;
```

Comment:
One note for @bernardkwok:  It looks like this changelist transforms ShaderRenderer::saveImage into a pure image-saving method, one that no longer references the contents of the renderer's off-screen buffer.  If that's the intent, then perhaps we should make it a simple helper function of the Image class, rather than a ShaderRenderer method.  What are your thoughts?